PowerPoint_Adapter/AdapterActions/Push.cs

```diff
@@ -97,7 +104,7 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
             try
             {
                 if (m_OutputFileSettings != null)
-                    presentationDoc.SaveAs(m_OutputFileSettings.GetFullFileName());
+                    presentationDoc.Clone(m_OutputFileSettings.GetFullFileName());
```

Comment:
```suggestion
                    presentationDoc.Clone(m_OutputFileSettings.GetFullFileName()).Dispose();
```
This is the fix that allows scripts that use the adapter to remain open if you want to open the output presentation, as the garbage collector doesn't dispose the cloned document automatically (for some reason).